### PR TITLE
ENG-24098-create-origins-messages-errors

### PR DESCRIPTION
**Pull request by gabriel-azion**
Origins: 
- errors
- messages

### CODE REVIEW

File: messages/origins/errors.go [hunk 1]
```diff
@@ -5,14 +5,13 @@ import "errors"
...
-	ErrorMandatoryUpdateFlags         = errors.New("A mandatory flag is missing. You must provide --application-id and --origin-id flags when the --application-id and --in flag is not provided. Run the command 'azioncli domains create --help' to display more information and try again.")
-	ErrorHmacAuthenticationFlag       = errors.New("Invalid --hmac-authentication flag provided. The flag must have  'true' or 'false' values. Run the command 'azioncli domains <subcommand> --help' to display more information and try again.")
+	ErrorMissingApplicationIDArgument = errors.New("A required flag is missing. You must supply application-id as an argument. Run 'azioncli <command> <subcommand> --help' command to display more information and try again")
+	ErrorMissingArguments             = errors.New("Required flags are missing. You must supply application-id and origin-id as arguments. Run 'azioncli <command> <subcommand> --help' command to display more information and try again")
```
Comment (PatrickMenoti) [inline messages/origins/errors.go:9]: We will have to update all messages with this new standard (future PRs)

File: messages/origins/errors.go [hunk 2]
```diff
@@ -5,14 +5,13 @@ import "errors"
...
+	ErrorMissingArgumentsDelete       = errors.New("Required flags are missing. You must supply application-id and origin-key as arguments. Run 'azioncli <command> <subcommand> --help' command to display more information and try again")
+	ErrorMandatoryCreateFlags         = errors.New("Mandatory flags are missing. You must provide application-id, name, addresses and host-header flags when the --application-id and --in flag are not provided. Run the command 'azioncli <command> <subcommand> --help' to display more information and try again.")
+	ErrorMandatoryUpdateFlags         = errors.New("Mandatory flags are missing. You must provide application-id and origin-id flags when the --application-id and --in flag are not provided. Run the command 'azioncli <command> <subcommand> --help' to display more information and try again.")
```
Comment (PatrickMenoti) [inline messages/origins/errors.go:12]: Should be either required or mandatory

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

File: messages/origins/errors.go (hunk 2 above)
Comment (gabriel-azion) [reply messages/origins/errors.go:12]: agreed

Comment (gabriel-azion) [reply messages/origins/errors.go]: done @PatrickMenoti

**Comment by gabriel-azion** [comment]
I have read the CLA Document and I hereby sign the CLA

File: messages/origins/messages.go [hunk 3]
```diff
@@ -1,77 +1,77 @@
...
+	OriginsUpdateShortDescription         = "Updates an Origin"
+	OriginsUpdateLongDescription          = "Updates an Origin based on its ID and given attributes"
+	OriginsUpdateFlagOriginKey            = "The Origin's key unique identifier"
+	OriginsUpdateFlagEdgeApplicationId    = "Unique identifier for an edge application"
+	OriginsUpdateFlagName                 = "The origin's name"
+	OriginsUpdateFlagOriginType           = `Identifies the source of a record. I.e. "single_origin"`
+	OriginsUpdateFlagAddresses            = "Passes a list of addresses linked to the origin"
+	OriginsUpdateFlagOriginProtocolPolicy = "Tells the protocol policy used in the origin"
+	OriginsUpdateFlagHostHeader           = "Specifies the hostname of the server being accessed"
+	OriginsUpdateFlagOriginPath           = "Gives a file path to the origin"
```
Comment (PatrickMenoti) [inline messages/origins/messages.go:57]: Origin Path from RTM: Azion can request your content from a directory in your origin. Azion appends Origin Path to the URI when forwarding the request to your origin. Leave it in blank to use only the URI.

**Review by PatrickMenoti — APPROVED** [review]
LGTM

**Pull request merged**